README.md

```diff
@@ -40,8 +41,8 @@ func main() {
         panic(err)
     }
 
-    if alice != aliceAgain {
```

Comment:
In the tests you're using `Expect(x).To(Equal(y))` which uses `reflect.DeepEqual` under the hood. You can't just use `==` in this case because the struct contains a slice.